tests/PHPUnit/System/BackwardsCompatibility1XTest.php

```diff
@@ -106,7 +106,8 @@ public function getApiForTesting()
             // those reports generate a different segment as a different raw value was stored that time
             'DevicesDetection.getOsVersions',
             'UserSettings.getOS',
-            'UserSettings.getBrowserType'
+            'UserSettings.getBrowserType',
+            'Goals.get'
```

Comment:
We need to execute `Goals.get` separate as here in this test `browserArchiving` is disabled. This means there will be never a value for the returning / new segments. Therefore the expected one is different to expected that we use usually.